pages/releases/4.3.html

```diff
@@ -0,0 +1,2109 @@
+										<code>
+											<span class="keyword">is not</span>
+										</code>
+										operator
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										<code class="highlight">
+											<span class="controlflow">if</span>
+											<span class="keyword">not</span>
+											my_node
+											<span class="keyword">is</span>
+											<span class="basetype">Node3D</span><span class="symbol">:</span>
+										</code>
+									</p>
+									<p>
+										It isn't much readable, huh? This line tests if
+										<code class="highlight">my_node</code>
+										is <em>not</em> of type
+										<code class="highlight">
+											<span class="basetype">Node3D</span>
+										</code>.
+									</p>
+									<p>
+										You can now type this instead, which is really closer syntax wise from what
+										we actually say and mean:<br>
+										<code class="highlight">
+											<span class="controlflow">if</span>
+											my_node
+											<span class="keyword">is not</span>
+											<span class="basetype">Node3D</span><span class="symbol">:</span>
+										</code>
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="dalexeev"
+									data-read-more="https://github.com/godotengine/godot/pull/87939">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #gdscript-is-not-operator -->
+
+				<div id="gdscript-built-in-methods-callable" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdscript-built-in-methods-callable">
+										Built-in methods and utility functions usable as
+										<code>
+											<span class="basetype">Callable</span>
+										</code>
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										You have an array and you want to print each element without creating
+										a for loop?
+									</p>
+									<p>
+										By making built-in methods and utility functions usable as
+										<code>Callable</code> makes it possible to call variadic functions such as
+										<code class="highlight">
+											<span class="gdscript-globalfunction">print</span><span class="symbol">()</span>
+										</code>
+										with
+										<code class="highlight">
+											<span class="basetype">Callable</span><span class="symbol">.</span><span class="function">callv</span><span class="symbol">()</span>
+										</code>
+										easily, without the need of boilerplate code.
+									</p>
+									<p>
+										You can now call this:<br>
+										<code class="highlight">
+											print<span class="symbol">.</span><span class="function">callv</span><span class="symbol">(</span>my_array<span class="symbol">)</span>
+										</code>
+									</p>
+								</div>
```

Comment:
```suggestion
								<h4 class="c-title">
									<a href="#gdscript-built-in-methods-callable">
										Built-in functions are now usable as
										<code>
											<span class="basetype">Callable</span>
										</code>
									</a>
								</h4>
								<div class="c-content">
									<p>
										This change affects both methods and utility functions. We made it so that when you try to call these functions as you would with a Callable, an internal conversion happens to allow that to work. 
									</p>
									<p>
								         For instance <code class="highlight"><span class="gdscript-globalfunction">print</span><span class="symbol">()</span></code>
								         Can now be called like so:
									<code class="highlight">print<span class="symbol">.</span><span class="function">callv</span><span class="symbol">(</span>my_array<span class="symbol">)</span></code>
									</p>
									<p> 
									This is particularly useful for variadic functions.
									</p>
								</div>
```